public/script.js

```diff
@@ -18,6 +18,7 @@ const myNickname = localStorage.getItem("nickname");
 const myGameRoomCode = localStorage.getItem("roomCode");
 const amIHost = localStorage.getItem("isHost");
 const startGameBtn = document.getElementById("btn-startgame");
+const soundControl = document.getElementById("sound");
```

Comment:
This variable name doesn't clarify what it holds. It should be `audioControlChkBox`